doc/source/skips/1-values.rst

```diff
@@ -0,0 +1,168 @@
+long enough and can move forward with formal adoption. Most comments were
+positive, so below I'll just summarize the “negative” comments under “rejected
+ideas”.
+
+Detailed description
+--------------------
+
+(Or: What problem does this proposal solve?)
+
+Over the past few years, scikit-image has been slightly “adrift”, with new and
+old contributors coming in, adding what small bits they need at the time, and
+disappearing again for a while. This is *fine* and we want to encourage more of
+it, but it also lacks direction. Additionally, without a concerted roadmap to
+concentrate effort, many of these contributions just fall by the wayside, as it
+is difficult for new contributors to live up to our stringent (and largely
+unwritten) standards of code.
+
+Implementation
+--------------
+
+Our mission
+***********
+
+scikit-image aims to be the essential toolkit for scientific image analysis in
+Python. We accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new
+  dependencies, and sometimes cull existing ones, or make them optional.
+- providing a **consistent API**. Conceptually identical arguments have the
+  same name and position in a function signature.
+- **ensuring accuracy**. Test coverage is close to 100% and code is reviewed by
+  at least two core developers before being included in the library.
+- **caring about users’ data**. We have a functional [2]_ API and don't modify
+  input arrays unless explicitly directed to do so.
+- promoting **education** (including self-education) **in image processing**,
+  with extensive pedagogical documentation.
+
+Our values
+**********
+
+- We are inclusive. We continue to welcome and mentor newcomers who are
+  making their first contribution.
+- We are community-driven. Decisions about the API and features are driven by
+  our users' requirements, not by the whims of the core team. (See
+  :ref:`governance`.)
+- We serve scientific applications primarily, over “consumer” image editing in
+  the vein of Photoshop or GIMP. This often means prioritizing n-dimensional
+  data support, and rejecting implementations of “flashy” filters that have
+  little scientific value.
+- We value elegant implementations over getting every last ounce of
+  performance. Readable code that is easy to understand, for newcomers and
+  maintainers alike, makes it easier to contribute new code as well as prevent
+  bugs. This means that we will prefer a 20% slowdown if it reduces lines of
+  code two-fold, for example.
+- We require excellent documentation. All functions should have NumPy-style
+  docstrings [3]_, preferably with doctest examples, as well as gallery
+  examples that showcase how that function is used in a scientific application.
+  Good documentation is hard, so this requirement can stall many contributions.
+  Core developers take an active role in finishing final documentation
+  examples.
+- We don't do magic. We use NumPy arrays instead of fancy façade objects
```

Comment:
I think this is covered by the point above about readable code.

Reply:
I don't think so at all. Magic can be simple and readable, such as checking whether the last dimension of an array has size 3. Ditto about façade objects. We had them historically because they *can* seem so elegant and readable at the time.